app/src/main/java/memphis/myapplication/data/tasks/RttEstimator.java

```diff
@@ -0,0 +1,171 @@
+/**
```

Comment:
This file and the SegmentFetcher file shouldn't be included in npChat. Your update has been merged into the jNDN library, so we should be using that.

Reply:
Replaced